### ISSUE TITLE
Fix issue with Vagrant and settings files.

### ISSUE THREAD

**Issue by brittainhard** [memex-explorer/memex-explorer#570]
Vagrant is still not properly copying the deploy settings file. Therefore, every time a nutch crawl is run there are failures. There needs to be a fix to this.

**Comment by brittainhard** [comment]
Current issue:

```
          ID: /vagrant/source/memex/settings.py
    Function: file.copy
      Result: True
     Comment: The target file "/vagrant/source/memex/settings.py" exists and will not be overwritten
     Started: 17:29:25.363369
    Duration: 6.775 ms
     Changes:  
```

**Comment by ahmadia** [comment]
This is a Salt, issue, not a Vagrant issue.  You can either force over-writing the file yourself (Salt will not, by default, overwrite a file when it exists), or you can tell Salt to always overwrite the file.  We need to have a more clear strategy for how we're using Salt in both deployment and development, but that's probably not a conversation to have in this specific issue, since issues should attempt to discuss one thing (whatever the title of the issue is).

**Comment by brittainhard** [comment]
Fixed in #8b0ad1d4981d2e43c5ea48e42a8dee6711e62f90
